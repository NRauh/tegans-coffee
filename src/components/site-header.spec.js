import React from 'react';
import renderer from 'react-test-renderer';
import SiteHeader from './site-header';

describe('SiteHeader', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SiteHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
