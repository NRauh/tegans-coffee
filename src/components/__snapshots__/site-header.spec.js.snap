// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteHeader renders correctly 1`] = `
<header
  css={
    Object {
      "backgroundColor": "#f5f5f5",
      "padding": "0.63rem 0.9rem",
    }
  }
>
  <h2
    css={
      Object {
        "fontSize": "1.35rem",
        "marginBottom": "0rem",
      }
    }
  >
    <a
      css={
        Object {
          "color": "#727272",
          "textDecoration": "none",
        }
      }
      href="/"
    >
      Tegan Rauh
    </a>
  </h2>
</header>
`;
